sled-agent/src/http_entrypoints.rs

```diff
@@ -280,3 +283,37 @@ async fn vpc_firewall_rules_put(
 
     Ok(HttpResponseUpdatedNoContent())
 }
+
+/// Create a mapping from a virtual NIC to a physical host
+#[endpoint {
+    method = PUT,
+    path = "/set-v2p",
+}]
```

Comment:
Strictly speaking the mapping we're applying is for a specific _network interface_, not instance. For example, during a `PUT /v1/network-interfaces/{interface}`, this would incur `PUT /v2p/{interface}` and `DELETE /v2p/{interface}` calls, but not affect the instance placement. I've done this in ffb47670

Reply:
Yes, you're right, the mappings tell OPTE where to find a virtual address in the physical world. And all NICs for an instance are on the same host and share the same VNI, so I was only hoping to simplify the interfaces. If you feel it doesn't do that, that's cool too.

I don't quite follow the bit about incurring some additional calls, could you expand on that?

Reply:
Sorry, I only mean that "moving" a NIC around (if possible) would incur calls to interface specific endpoints, not instance specific endpoints.